### ISSUE TITLE
reconsider timeline for 1.0 release

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/graphing-quadratics#101]
If wrapping up related PhET-iO work doesn't push us out to far, it would be desirable to publish with `brands=phet,phet-io`. So let's reconsider the timeline for 1.0 release.  The currently timeline includes completing code review and starting RC testing during the week of 11/5/18.

<details>
<summary>Slack conversation with @amanda-phet and @ariel-phet</summary>

11/2/18:

Chris Malley [9:51 AM]
The PhET-iO team has been working hard to address PhET-iO issues, so that GQ 1.0 can include brand=phet-io.  Work probably won’t be completed in order to start RC testing next week, but might be ready if we push RC testing out a bit.  How much of a slip in the GQ 1.0 schedule would be acceptable? What is our “drop dead” date for starting RC testing?

Amanda McGarry [10:08 AM]
I don't personally have a drop dead date-- I assumed it was all based on the gas properties schedule. At one point I said it would be nice to be published by 11/13/18 but that is certainly not a requirement. So, do what you need to do. I think it's great to have phet-io done now so you don't have to come back to it later!

Ariel Paul [11:08 AM]
The only "drop dead" date of which I am aware is published by Jan 31
Is the November date for a workshop? We could certainly use a stable dev version for that (not ideal I know, but totally doable)

Chris Malley [3:54 PM]
Thanks. I’ll discuss with the PhET-iO team and propose a timeline.
</details>

**Comment by pixelzoom** [comment]
@samreid @zepumph when do you anticipate that all blocking PhET-iO work will be completed?

**Comment by samreid** [comment]
We have a PhET-iO mini meeting scheduled for Wednesday Nov 7.  Hopefully by the end of that meeting we can give a reasonable time estimate.

**Comment by zepumph** [comment]
Currently searching for https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+user%3Aphetsims+label%3Adev%3Aphet-io+label%3Astatus%3Ablocks-publication yields 7 issues. 2 do not apply to GQ, 3 are up for review (already implemented), and 2 need work still. Potentially much of the work on the final two issues can be done on Wednesday. I agree that we will know more by Wednesday, but I wanted to give this more specific update as well.

**Comment by samreid** [comment]
We made a lot of progress today, perhaps we can touch base at 10am PhET-iO meeting on Wed Nov 8 and make sure everything is all set.

**Comment by pixelzoom** [comment]
11/8/18 phet-io meeting notes:

All phet-io issues are in a state that we could use for GQ 1.0, so proceed.

We'll do things in this order:
* memory leak testing for brands=phet-io, see #50
* dev test brands=phet-io
* RC test brands=phet,phet-io

**Comment by pixelzoom** [comment]
We are now blocked by https://github.com/phetsims/phet-io/issues/1411.

**Comment by pixelzoom** [comment]
1.0 release (and further work on #52) is now blocked by https://github.com/phetsims/axon/issues/197, assigned to @jonathanolson.

**Comment by jonathanolson** [comment]
https://github.com/phetsims/axon/issues/197 should be resolved.

**Comment by pixelzoom** [comment]
After today's PhET-iO dev call with @kathy-phet, the decision was made to rollout Graphing Quadratics 1.0 with the redesigned Studio.  So that will push the schedule out further.  No idea when we'll start the QA process. The way things are going, I'll be pleasantly surprised if this sim is published before the end of the year. @ariel-phet @amanda-phet FYI.

**Comment by kathy-phet** [comment]
@pixelzoom - It sounded like a decision had already been made to publish PhET-iO 1.0 at the same time as graphing quadratics regular. We decided that we should at least fix the hierarchy in the sim, adding the .general. code structure before publishing so the PhET-iO version wouldn't have that breaking, but I don't think  all of PhET-iO Studio should block publication.

**Comment by pixelzoom** [comment]
Publishing brand=phet and brand=phet-io 1.0 at the same time was a goal from the beginning of development. When the list of phet-io issues was long, we briefly considered publishing brand=phet only. When progress was made on PhET-iO issues, we reconsidered. And that reconsideration is the topic of this issue, from line 1 of the first comment:

> If wrapping up related PhET-iO work doesn't push us out to far, it would be desirable to publish with brands=phet,phet-io. 

That was 24 days ago, and we've started to diverge again.  We've added some portion of a redesigned Studio, and instrumentation of "features" components.  I agree these are big improvement. And if including it works for PhET scheduling, great.  But considering that a major holiday is a little more than 3 weeks away, I don't feel like I'm being too pessimistic when I say that I'll be surprised if we publish this sim before the end of the year.

**Comment by pixelzoom** [comment]
In 11/29/18 status meeting, @kathy-phet listed all of the sims that need to be published for the January deadline, and all of the other sims that are in the process of being republished.  We clearly have a logjam coming up in January.  So @kathy-phet made the decision that we will not publish a PhET-iO version for Graphing Quadratics 1.0. #115 (review PhET-iO API) can be deferred, dev testing is not necessary (since we've already dev tested brand=phet) and we should proceed immediately to RC testing.

Closing.

**Comment by pixelzoom** [comment]
In 11/29/18 status meeting, @kathy-phet listed all of the sims that need to be published for the January deadline, and all of the other sims that are in the process of being republished.  We clearly have a logjam coming up in January.  So @kathy-phet made the decision that we will not publish a PhET-iO version for Graphing Quadratics 1.0. #115 (review PhET-iO API) can be deferred, dev testing is not necessary (since we've already dev tested brand=phet) and we should proceed immediately to RC testing.

@amanda-phet @ariel-phet @samreid @chrisklus @zepumph FYI.

Closing.
